### DISCUSSION TITLE
Store several specific output into *.e format

### DISCUSSION THREAD

**Discussion by hokonkim** [idaholab/moose#20421] [Q&A General] [answered]
Hello!

I have a question about storing the result file (*.e).

After the simulation, I want to store all output value into csv.

But simultaneously I would like to store the result for several specific time step.

e.g, 10 time steps.
      All output value for 10 time steps.
      5 results for every 2 step.

Is this possible? If yes, how can I get this?

I always appreciate your help.

Sincerely,
Hokon Kim

**Comment by GiudGiud** [comment, marked as answer]
Hello

Just create more than one CSV block in [Outputs].

```
[Outputs]
  [csv_all]
    type = CSV
  []
  [csv_others]
    type = CSV
# use start/end  _step (or _time) + interval to do what you want

  []
[]
```

**Reply by GiudGiud** [reply]
Note that step 1-10 then every five steps might not be possible with just the parameters we have now. You ll have to either change those parameters (not too hard coding), create a custom EXECUTE_ON (also not too hard coding), OR just have 3 CSV blocks and merge two of them afterwards
